plugins/outputs/zabbix/zabbix.go

```diff
@@ -60,7 +60,7 @@ func (*Zabbix) SampleConfig() string {
 // Connect does nothing, Write() would initiate connection in each call.
 // Checking if Zabbix server is alive in this step does not allow Telegraf
 // to start if there is a temporal connection problem with the server.
-func (z *Zabbix) Connect() error {
+func (*Zabbix) Connect() error {
```

Comment:
Given that both `Connect` and `Close` are empty, we can remove both, can't we?

Reply:
@srebhan Then this plugin would stop being an output plugin:
![image](https://github.com/user-attachments/assets/b334508d-8a55-4ed9-bb42-319ca3781e67)